service/orchestrator/catalogs.go

```diff
@@ -220,23 +224,42 @@ func (svc *Service) loadCatalogs() (err error) {
 		log.Errorf("Error while saving catalog %v", err)
 	}
 
-	log.Debugf("Catalog loaded with id '%s'.", catalogs[0].GetId())
+	log.Debug("Catalogs successfully stored.")
 
 	return
 }
 
 func (svc *Service) loadEmbeddedCatalogs() (catalogs []*orchestrator.Catalog, err error) {
-	var b []byte
+	var (
+		b        []byte
+		fileList []string
+	)
 
-	log.Infof("Loading catalogs from %s", svc.catalogsFile)
-	b, err = f.ReadFile(svc.catalogsFile)
-	if err != nil {
-		return nil, fmt.Errorf("error while loading %s: %w", svc.catalogsFile, err)
-	}
+	// Get all filenames
+	fileList, err = util.GetJSONFilenames(svc.catalogsFolder)
 
-	err = json.Unmarshal(b, &catalogs)
-	if err != nil {
-		return nil, fmt.Errorf("error in JSON marshal: %w", err)
+	log.Infof("Loading catalogs from files %s.", strings.Join(fileList, ", "))
+
+	// Get catalog for each file
+	for i := range fileList {
+		var catalogsFromFile []*orchestrator.Catalog
+
+		// Get current path
+		path, _ := os.Getwd()
```

Comment:
This should be unnecessary, I am pretty sure you can just use a relative path in `os.Readfile`, so no need to join it with the current directory.